src/routes.json

```diff
@@ -247,33 +258,40 @@
         "title": "Scratch Terms of Use"
     },
     {
+        "name": "things-to-try",
+        "pattern": "^/go/?(\\?.*)?$",
+        "routeAlias": "/go/?\\??",
+        "redirect": "/tips"
+    },
```

Comment:
Since we replaced the Things to Try page with Tips, this route is now a redirect. It should probably be with the redirects (and probably renamed things-to-try-redirect). There are a couple of others that have also been converted into redirects `hoc`, `help`. @mewtaylor do you think we should rename them `-redirect` and move them to the redirects section?